Samples/MobileBuyIntegration/MobileBuyIntegration.xcodeproj/project.pbxproj

```diff
@@ -307,28 +304,11 @@
 			outputFileListPaths = (
 			);
 			outputPaths = (
+				"${TARGET_BUILD_DIR}/${FULL_PRODUCT_NAME}/.cleanup_complete",
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "# Remove .pcm binaries from build\nfind \"${TARGET_BUILD_DIR}/${FULL_PRODUCT_NAME}\" -name \"*.pcm\" -type f -delete\n\n# Remove Swiftlint binaries from build\nfind \"${TARGET_BUILD_DIR}/${FULL_PRODUCT_NAME}\" -iname \"swiftlint*\" -type f -delete\n\n# Remove .dev directory from build\nfind \"${TARGET_BUILD_DIR}/${FULL_PRODUCT_NAME}\" -name \".dev\" -type d -exec rm -rf {} +\n\n\n\n";
-		};
-		6A9124C62CF6114E0076C21C /* Run Swiftformat over sample code */ = {
-			isa = PBXShellScriptBuildPhase;
-			buildActionMask = 2147483647;
-			files = (
-			);
-			inputFileListPaths = (
-			);
-			inputPaths = (
-			);
-			name = "Run Swiftformat over sample code";
-			outputFileListPaths = (
-			);
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "export PATH=\"/opt/homebrew/bin:$PATH\"\n\nif [ \"$CI\" = \"true\" ]; then\n  echo \"Skipping SwiftFormat in CI environment\"\n  exit 0\nfi\n\nif which swiftformat >/dev/null; then\n\tswiftformat \"${SRCROOT}\" \nelse  \n\techo \"error: SwiftFormat not installed, download using \\\"brew install swiftformat\\\"\"\nfi\n";
```

Comment:
We don't want to do this anymore since there are side effects like not being able to `CMD + Z` after building